client/src/ui/organisms/footer.tsx

```diff
@@ -1,22 +1,21 @@
 import * as React from "react";
 
-import { useLocale } from "../../hooks";
-import Logo from "../atoms/logo";
-
 import { ReactComponent as GithubIcon } from "@mdn/dinocons/brands/github-mark-small.svg";
 import { ReactComponent as TwitterIcon } from "@mdn/dinocons/social/twitter.svg";
 import { ReactComponent as InstagramIcon } from "@mdn/dinocons/social/instagram.svg";
 
+import { useLocale } from "../../hooks";
+import { Logo } from "../atoms/logo";
```

Comment:
Perhaps a confusing diff, but the local imports should come *after* the third-party imports.